data/campaigns/Winds_of_Fate/scenarios/05_The_Three_Sisters.cfg

```diff
@@ -0,0 +1,840 @@
+                description= _ "Capture all villages to secure a supply of food before the arrival of our Flight."
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Reshan"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Arinexis"
+                condition=lose
+                [show_if]
+                    [have_unit]
+                        id=Arinexis
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Death of Zedrix"
+                condition=lose
+                [show_if]
+                    [have_unit]
+                        id=Zedrix
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            {TURNS_RUN_OUT}
+            [gold_carryover]
+                carryover_percentage=0
+            [/gold_carryover]
+            [note]
+                description= _ "The Flight has yet to arrive. Recruiting will not be possible in this scenario. Neither will recalling veterans."
+            [/note]
+            [note]
+                description= _ "The native beasts of these isles have little interest in the villages we seek to secure.
+Yet there may be attempts to recapture villages by the invasive haldric creatures (humans)."
+            [/note]
+        [/objectives]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Gorlack
+            message= _ "It is here I will plant my eyrie.
+
+This will be the threshhold through which our folk will pass.
+From our long shameful exile adrift on sinking isles.
+Returned to our forgotten birthright of this greater world."
+        [/message]
+        [message]
+            speaker=Reshan
+            message= _ "The game looks not as numerous as these isles should bear...
+The landbound creatures may have infested here also."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "Then we shall seize their settlements to procure our Flight's sustenance."
+        [/message]
+        [message]
+            speaker=Reshan
+            message= _ "There is some commotion below us...
+
+Hatchlings!  There are hatchlings just below. They look to be in dire need of aid!"
+        [/message]
+        [scroll_to]
+            location_id=2
+        [/scroll_to]
+        [delay]
+            time=1500
+        [/delay]
+        [message]
+            speaker=Rhynnin
+            image_pos=right
+            mirror=yes
+            message= _ "Excellent, we have them surrounded once more. This time do not allow any to slip through; the King will have everyone of their green heads for his mantle.
```

Comment:
"every one" instead of "everyone"?

Reply:
Yes, should be 'every [single] one' in this context.

Reply:
Nice catch, thanks!